### PR TITLE
[YP2-18] Fix errors related to W3C validator.

### PR DESCRIPTION
**Pull request by manachynskyi**
This PR is going to fix some W3C html errors for Carnation theme.
List of errors https://validator.w3.org/nu/?showsource=yes&showoutline=yes&doc=https%3A%2F%2Fwww.develop-sr3snxi-hmd35zkxohky4.us-3.platformsh.site%2F#l656c60

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/openy_carnation.libraries.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ global-styling:
   version: 1.x
   css:
     base:
-      //use.fontawesome.com/releases/v5.1.0/css/all.css: {type: external, attributes: { defer: true, async: true} }
+      //use.fontawesome.com/releases/v5.1.0/css/all.css: {type: external}
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/openy_carnation.libraries.yml:5]: why? this was performance tuning

File: themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig [hunk 2]
```diff
@@ -59,19 +59,20 @@
     <div class="banner-cta-content container align-items-center m-auto py-5 text-center text-white">
       <div class="banner-cta-section">
 
-        {% if not content.field_prgf_headline.isEmpty() %}
+        {% if not paragraph.field_prgf_headline.isEmpty() %}
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig:62]: this is not a W3C related refactoring. Let's avoid having unrelated changes for BC

**Review by podarok — CHANGES_REQUESTED** [review]

File: themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig (hunk 2 above)
Comment (manachynskyi) [reply themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig:62]: Yes, it is not related to W3C, but I noticed this bug (content.field_prgf_headline is an array and we can't perform isEmpty on it). Should it be done in separate PR?

File: themes/openy_themes/openy_carnation/openy_carnation.libraries.yml (hunk 1 above)
Comment (manachynskyi) [reply themes/openy_themes/openy_carnation/openy_carnation.libraries.yml:5]: defer and async don't work for css, only for scripts. 
first request with redirect https://i.imgur.com/ORHixFG.png
second https://i.imgur.com/uLvXcmx.png

https://www.w3schools.com/tags/att_defer.asp
https://www.w3schools.com/tags/att_async.asp

**Review by dchubar-fj — APPROVED** [review]

File: themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig (hunk 2 above)
Comment (podarok) [reply themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig:62]: let it be here, thank you
